src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -458,6 +458,10 @@ input.inline {
     width: 60%;
 }
 
+.panel-horizontal {
+    display: inline-block;
```

Comment:
This rule is set to prevent cases where the spreadsheet interface go out of the panel when it is stretched too much:

![image](https://user-images.githubusercontent.com/25320782/40414157-1c811df2-5eaa-11e8-9065-2c59fa30ec61.png)

With this rule, the panel stretches along with the spreadsheet interface:

![may-23-2018 16-59-20](https://user-images.githubusercontent.com/25320782/40414416-b581d604-5eaa-11e8-88fd-7053f21616c6.gif)

Reply:
I think a better way is to make only the spreadsheet horizontally scrollable if its width exceeds the allowed size.

Reply:
I experimented with two ways of making the spreadsheet horizontally scrollable if its width exceeds the allowed size (I removed the `inline-block` style when doing this):

1. Handsontable's built in `preventOverflow: 'horizontal'` setting
2. Putting this rule in the css file:
```
#spreadsheet {
    width: auto;
    overflow-x: scroll;
    overflow-y: hidden;
}
```
Both of them produce the similar anomaly:

1. When the user manually enters new rows through keystrokes
![horizontal-scroll-manual](https://user-images.githubusercontent.com/25320782/40525738-80b070a0-6014-11e8-87bc-b4a07bc2b200.gif)

2. When the user adds new rows through the `Add` button
![horizontal-scrolling-button](https://user-images.githubusercontent.com/25320782/40525742-9051ac7c-6014-11e8-8f8f-675896c84715.gif)

This seems like an issue from the handsontable side that we have minimal control over. This was part of the reason why I thought of separating functionalities of the spreadsheet interfaces in different pages as addressed by my comment in #8832.

Reply:
I don't understand the `    overflow-y: hidden;` rule. This means if the height exceeds the allowed size, its content will be hidden. That's not what we want right? I think we should either make the spreadsheet vertically scrollable, or just don't do impose a limit on the height.

Reply:
Initially I tried `overflow-y: scroll` but the UI turns out to be a little wonky. I experimented new changes and it seems to achieve a much better user experience.

It consists of:
1. Handsontable's built in preventOverflow: 'horizontal' setting
2. New rule:
```
.htCenter {
    overflow-y: auto;
    height: 200px;
}
```
A min height needs to be set for `overflow-y` to work and Handsontable does not support `preventOverflow: 'vertical'` yet, thus the reason for this additional css rule.

Any thoughts on the new UX?
![new-ux](https://user-images.githubusercontent.com/25320782/40532303-9b631e76-6031-11e8-96d4-cdbe888a8fd8.gif)

Reply:
The behaviour is good. But I think the vertical ratio between different elements is not very reasonable. Like there is a big gap between the "If you want to..." message and the table and the 200px height is a little bit too little for this region which is the main element that the user interact with in this page.